Copyright 2003, Robin H. Johnson <robbat2@orbis-terrarum.net>, All rights
reserved.
This software is not subject to any license of Gentoo Technologies, Inc. or
Simon Fraser University.

This is free software; you can redistribute and use it
under the same terms as OpenLDAP itself.
diff -ur openldap-2.1.22.orig/configure.in openldap-2.1.22/configure.in
--- openldap-2.1.22.orig/configure.in	2003-06-21 10:18:37.000000000 -0700
+++ openldap-2.1.22/configure.in	2003-10-12 14:14:42.000000000 -0700
@@ -646,10 +646,11 @@
 
 	else
 		PERL_CPPFLAGS="`$PERLBIN -MExtUtils::Embed -e ccopts`"
+		PERL_LDFLAGS="`$PERLBIN -MExtUtils::Embed -e ldopts|sed -e 's/\W-lc\>//g;s/^-lc\>//'`"
 		if test x"$ol_with_perl_module" = "xstatic" ; then
-			SLAPD_PERL_LDFLAGS="`$PERLBIN -MExtUtils::Embed -e ldopts|sed -e s/-lc//`"
+			SLAPD_PERL_LDFLAGS="$PERL_LDFLAGS"
 		else
-			MOD_PERL_LDFLAGS="`$PERLBIN -MExtUtils::Embed -e ldopts|sed -e s/-lc//`"
+			MOD_PERL_LDFLAGS="$PERL_LDFLAGS"
 		fi
 		dnl should check perl version
 		ol_link_perl=yes
